index.py

```diff
@@ -34,19 +35,17 @@
     You may close it after the bot has been invited and the command has been ran{Style.RESET_ALL}
 """))
 
-# Try except block is useful for when you'd like to capture errors
-try:
+# Checks if a config file has been made
+# If so, load said config, else load no config
+if path.isfile("config.json"):
     with open("config.json") as f:
         config = json.load(f)
-except (FileNotFoundError, json.JSONDecodeError):
-    # You can in theory also do "except:" or "except Exception:", but it is not recommended
-    # unless you want to suppress all errors
+else:
     config = {}
```

Comment:
The reason this was a try/except to begin with is because it checks for 2 errors in one go, if JSON file has been corrupted or if file was not found, making the code minimized. Unless you have a better solution for this checking, I'd probably suggest reverting this change